src/MiniCover/HitServices/Hit.cs

```diff
@@ -1,54 +1,29 @@
-﻿
-namespace MiniCover.Reports
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MiniCover.HitServices
 {
-    internal abstract class Hit
+    [Serializable]
+    public class Hit
     {
-        public int InstrumentationId { get; }
-        public int Counter { get; }
-
-        protected Hit(int instrumentationId, int counter)
-        {
-            InstrumentationId = instrumentationId;
-            Counter = counter;
-        }
+        public int InstructionId { get; set; }
 
-        internal static Hit Parse(string line)
-        {
-            var sections = line.Split(':');
-            if(sections.Length == 2)
-                return new HitOnly(int.Parse(sections[0]), int.Parse(sections[1]));
-            return new HitWithTestsInformation(int.Parse(sections[0]), sections[1], sections[2], sections[3], sections[4], int.Parse(sections[5]));
-        }
+        public int Counter { get; set; }
 
+        public HashSet<TestMethodInfo> TestMethods { get; private set; } = new HashSet<TestMethodInfo>();
```

Comment:
I'm adding a list to testmethods, this reduces a lot the output file.
The structure would be something like:
```json
{
   instruction: 1234,
   counter: 10, //(hits)
   testMethods: [
       { name: "TestA" },
       { name: "TestB" }
   ]
}
```